hackclub.com.yaml

```
@@ -1009,6 +1009,16 @@ dash.hackathons:
   ttl: 600
   type: A
   value: 5.161.160.54
+_acme-challenge.dash.hackathons:
+- ttl: 600
+  type: NS
+  value: hydrogen.ns.hetzner.com.
+- ttl: 600
+  type: NS
+  value: helium.ns.hetzner.de.
+- ttl: 600
+  type: NS
+  value: oxygen.ns.hetzner.com.
```

```suggestion
- ttl: 600
  type: NS
  values: 
    - hydrogen.ns.hetzner.com.
    - helium.ns.hetzner.de.
    - oxygen.ns.hetzner.com.
```

My bad, this is what I get for code reviewing early in the morning

---

Ooh, love clean DSL optimizations